addons/mail/static/src/models/rtc_configuration_menu.js

```diff
@@ -40,10 +40,10 @@ registerModel({
             this.userSetting.setAudioInputDevice(value);
         },
         /**
-         * @param {String} value
+         * @param {MouseEvent} ev
          */
-        onChangeThreshold(value) {
-            this.userSetting.setThresholdValue(parseFloat(value));
+        onChangeThreshold(ev) {
+            this.userSetting.setThresholdValue(parseFloat(ev.target.value));
```

Comment:
I usually prefer `Number` to `parseFloat`.
Consider the following example:
```js
Number("1337Odoo") // NaN
parseFloat("1337Odoo") // 1337
```
If some garbage value happened to end up there, I wouldn't expect it to be treated as a perfectly valid input, that's why I prefer the behaviour of `Number`.

In our case though, this is not a big deal, as it's unlikely that the value would be anything but a float.

Reply:
Yes, it's no big deal, as `ev.target.value` is controlled as being a float value, albeit in string form so need type-casting.

Eventually business code will not have typing, as everything will be replaced to models. There won't be any need for type-casting at all.